lib/jekyll-gist/gist_tag.rb

```diff
@@ -47,6 +47,7 @@ def determine_arguments(input)
       end
 
       # rubocop:disable Style/PreferredHashMethods
+      # Remove the check for `:has_key?` when the plugin drops support for `< Liquid 4.0`
```

Comment:
As https://pages.github.com/versions/  supports it, I'm fine with requiring Liquid 4.0.
That's up to someone from GitHub to decide.
/cc  @parkr

Reply:
Yeah, requiring Liquid 4 is fine by me, but give a major bump to this gem if you do that. For now, backwards compatibility is good.